doc/tutorials/inter-vm_communication.rst

```diff
@@ -0,0 +1,490 @@
+.. _inter-vm_communication:
+
+ACRN Inter-VM Communication
+##############################################
+
+ACRN supports three kinds of Inter-VM communication – Inter-VM vUART,
+Inter-VM network communication, and Inter-VM shared memory device (ivshmem).
+Each communication has its advantage and shortage and is listed below:
+
+Inter-VM vUART
+**************
+
+Inter-VM vUART communication is based on the vUART implementation. It is
+used to transfer data between two VMs at low speed. (Refer to :ref:`vuart_virtualization`
+and :ref:`vuart_config`).
+
+:numref:`Inter-VM vUART communication` shows the Inter-VM vUART communication overview:
+
+.. figure:: images/Inter-VM_vUART_communocation_overview.png
+   :align: center
+   :name: Inter-VM vUART communication
+
+   Inter-VM vUART communication
+
+- Pros:
+   - POSIX APIs.
+   - UART driver is available in most VM, development-friendly(POSIX can be called directly
+     to realize vUART communication without additional development drivers).
+
+- Cons:
+   - The communication rate is low.
+   - Need to configure communication vUART in hypervisor scenario.
+
+Inter-VM network communication
+******************************
+
+Inter-VM network communication is based on the network stack. ACRN supports
+both pass-through NICs to VM and Virtio-Net solutions. (Refer to :ref:`virtio-net`
+background introductions of ACRN Virtio-Net Architecture and Design). 
+
+:numref:`Inter-VM network communication` shows the Inter-VM network communication overview:
+
+.. figure:: images/Inter-VM_network_communication_overview.png
+   :align: center
+   :name: Inter-VM network communication
+
+   Inter-VM network communication
+
+- Pros:
+   - Socket-based APIs; development friendly.
```

Comment:
Same comment here, it would be good to rephrase somehow to indicate this is used via the classical socket API (and hence is a well know mechanism for many developers).

@dbkinder @amyreye , could you perhaps suggest some text to that effect?